master/static/srv/ship-logs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Ideally, we'd like to make ship_logs.py the entrypoint in task containers, so
+# it could capture all logs from any process in the process tree.
+#
+# But we can't actually set it as the entrypoint because we don't know how to
```

Comment:
It runs in a user image, and we can't know how to call `python3` inside arbitrary images.

Sometimes it's just called `python`.  Or maybe in some pathological image it isn't on the PATH and you have to invoke it with a full /path/to/python3.  In those cases the user can specify DET_PYTHON_NAME or whatever so that our system works.

I think I've only seen this come up once with real users.  But it has come up.

Reply:
It'd be useful, yes.  Probably something support would be interested in; a lot of the environment variables are pretty obscure for end users, and have only been useful in one or two situations.